source/TimePicker.js

```diff
@@ -470,10 +492,11 @@
 					if (this.meridiemEnable === true) {
 						this.createComponent(
 							{classes: 'moon-date-picker-wrap', components:[
-								{kind:'moon.MeridiemPicker', name:'meridiem', classes:'moon-date-picker-field', value: valueHours > 12 ? 1 : 0, meridiems: this.meridiems || ['am','pm'], onChange: 'meridiemPickerChanged'},
+								{kind:'moon.MeridiemPicker', name:'meridiem', classes:'moon-date-picker-field', meridiems: this.meridiems || [{name: 'AM', start: '00:00', end: '11:59'}, {name: 'PM', start: '12:00', end: '23:59'}], max: this.meridiems.length - 1 || 1, onChange: 'meridiemPickerChanged'},
```

Comment:
Hi @DavidUM, there is no `meridiems` property on this kind by default, so among other things, this will not work if `ilib` is not present (as the `ilib` initialization sets the `meridiems` property). We either need more guard code, or should set the initial value of `meridiems` to the defaults you added here.